src/main/kotlin/calculator/Operator.kt

```diff
@@ -1,22 +1,18 @@
 package calculator
 
 enum class Operator(val symbol: String, private val operation: IntOperation) {
-    PLUS("+", IntOperation { number1, number2 -> number1 + number2 }),
-    MINUS("-", IntOperation { number1, number2 -> number1 - number2 }),
-    MULTIPLY("*", IntOperation { number1, number2 -> number1 * number2 }),
-    DIVIDE("/", IntOperation { number1, number2 -> number1 / number2 });
+    PLUS("+", { number1, number2 -> number1 + number2 }),
+    MINUS("-", { number1, number2 -> number1 - number2 }),
+    MULTIPLY("*", { number1, number2 -> number1 * number2 }),
+    DIVIDE("/", { number1, number2 ->
+        if (number2 == 0) throw IllegalArgumentException("0으로 나눌 수 없습니다")
+        else number1 / number2
```

Comment:
꼭 else를 쓸 필요는 없는 것 같아요. 😃
[early return이란?](https://dncjf64.tistory.com/300)

```suggestion
        number1 / number2
```